website/docs/user-group-role/roles/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: About roles
+---
+
+import DocCardList from "@theme/DocCardList";
+import { useCurrentSidebarCategory } from "@docusaurus/theme-common";
+
+Roles are a way to simplify the assignment of permissions. Roles are also the backbone of role-based access control (RBAC), an industry standard for managing [access control](../access-control). In authentik, RBAC is how you manage access to system components and specific objects such as flows, stages, and users.
```

Comment:
```suggestion
Roles are a way to simplify the assignment of permissions. Roles are also the backbone of role-based access control (RBAC), an industry standard for managing [access control](../access-control). In authentik, RBAC is how you manage access to system components and specific objects such as flows, policies, etc.
```

I prefer leaving the example lists open-ended as the permissions apply to all objects

Reply:
I added the "etc." but did want to put policies since no where do we cover policy permissions on the docs. We can add it later, I guess...